examples/setting/Cargo.toml

```diff
@@ -14,4 +14,4 @@ no-default-features = true
 [dependencies.nokhwa]
 path = "../../../nokhwa"
 # EDIT THIS!
-features = ["input-v4l"]
+features = ["input-msmf"]
```

Comment:
This shouldn't be changed. It is expected that the user change it.

Reply:
right... my mistake

Reply:
It seems you closed the 0.9.2 branch merging with senpai. I'll wait for a running version of the senpai branch to study these problems again.
By the way... Just published my project using your library: https://github.com/quartzo/webcam-visca-ip